Show/ShowMapper.php

```diff
@@ -114,6 +114,47 @@ public function remove($key)
     }
 
     /**
+     * Removes a group.
+     *
+     * @param string $group          The group to delete
+     * @param string $tab            The tab the group belongs to, defaults to 'default'
+     * @param bool   $deleteEmptyTab Whether or not the Tab should be deleted, when the deleted group leaves the tab empty after deletion
```

Comment:
- the Tab => the parent tab
- "should be deleted" => "should be deleted too"
- it is ok to use linebreaks in comments, like this : 

```
* @param bool   $deleteEmptyTab Whether or not the parent tab should be deleted too, 
*                                                       when the deleted group leaves the tab empty after deletion
```